src/unifiedreport/agent/reportStatic.php

```diff
@@ -160,30 +171,39 @@ function assessmentSummaryTable(Section $section, $otherStatement)
     $table->addCell($cellFirstLen)->addText(htmlspecialchars(" "), $leftColStyle, "pStyle");
     $table->addCell($cellLen)->addText(htmlspecialchars(" "), $rightColStyleBlue, "pStyle");
 
+    $nocriticalfiles = "no critical files found, source code and binaries can be used as is";
```

Comment:
For all these strings a whitespace should be added at the beginning. Otherwise the checkboxes touch the text.